omod/src/test/java/org/openmrs/Allergen.java

```diff
@@ -0,0 +1,16 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs;
+
+//TODO this dummy class is just for the sake of making LatestObsRestControllerTest pass.
+//it fails because the platform dependency version currently used, does not have this class.
+//after this has been solved, this dummy class can be deleted.
```

Comment:
Maybe these should link to a JIRA ticket?

Reply:
I had forgotten to add the JIRA ticket in the PR description